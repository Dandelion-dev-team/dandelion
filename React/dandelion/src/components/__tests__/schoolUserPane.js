import React from "react"
import renderer from "react-test-renderer"
import SchoolUserPane from "../schoolUserPane"


describe("Project", () => {
    let originalFetch;
    const testProp = { 'school_id': '1', 'school_class': 'Test class', 'project': '1', "description": "test desc", "experiment": "Experiment", "hypotheses": "Test hypotheses", "variables": "test variables" };

    it("renders correctly", () => {
        const tree = renderer
            .create(<SchoolUserPane dataProp={testProp}/>)
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})

