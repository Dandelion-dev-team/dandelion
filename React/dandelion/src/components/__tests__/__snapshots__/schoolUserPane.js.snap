// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project renders correctly 1`] = `
<div>
  <div>
    <div
      className="title"
    >
      <h2>
        Student 
        1
         - 
        Test class
      </h2>
      <h2>
        1
         Project 
      </h2>
      <h3>
        Experiment
         Experiment
      </h3>
    </div>
    <div
      className="project-column"
    >
      <p>
        Description
      </p>
      <div
        className="description-box"
      >
        <p>
          test desc
        </p>
      </div>
      <p>
        Hypotheses
      </p>
      <div
        className="description-box"
      >
        <p>
          Test hypotheses
        </p>
      </div>
      <p>
        Variables
      </p>
      <div
        className="description-box"
      >
        <p>
          test variables
        </p>
      </div>
    </div>
  </div>
</div>
`;
